### DISCUSSION TITLE
passcode에서 -500 에러가 나는데 저만그런가요?

### DISCUSSION THREAD

**Discussion by wkswkfkwks** [storycraft/node-kakao#761] [General]
인증이 안되는데 저만 그런가요???

**Comment by wkswkfkwks** [comment]
보안인증은 날라오는데 인증하면 -500에러가 떠욤..ㅠㅠ
업데이트가 있었나봐요

**Reply by wkswkfkwks** [reply]
로그인에서는 -100이 뜨고...후하..

**Comment by pupuarw** [comment]
저도 똑같은 증상입니다

**Reply by pupuarw** [reply]
노드만 막힌것으로 확인은 했는데 우회할 방법이 있긴한거같습니다

**Reply by wkswkfkwks** [reply]
어떤 방법이 있나여ㅛ?

**Reply by pupuarw** [reply]
저도 방법은 찾고있는데 쉽지가않네요 노드만 막힌거면 다른쪽으로 진행을 해야할지

**Reply by wkswkfkwks** [reply]
흐미... 저도 해결되면 공유하겠습니다.

**Comment by humanbot00** [comment]
api.registerDevice({ email: "", password: "" }, passcode, true); forced빼면 500 안뜨고 잘됩니다

**Reply by wngks1232** [reply]
우오 되네요!!! 감삼다!!
